starboard/nplb/media_buffer_test.cc

```diff
@@ -318,12 +318,12 @@ TEST(SbMediaBufferTest, StorageType) {
   }
   SB_NOTREACHED();
 }
-#endif  // SB_API_VERSION < 16
 
 TEST(SbMediaBufferTest, UsingMemoryPool) {
   // Just don't crash.
   SbMediaIsBufferUsingMemoryPool();
```

Comment:
Consider leaving the function in SB16 so we get a chance to explicitly fail implementations returning false.